components/SettingsControls/src/SettingsCard/SettingsCard.Properties.cs

```diff
@@ -69,6 +68,15 @@ public partial class SettingsCard : ButtonBase
         typeof(SettingsCard),
         new PropertyMetadata(defaultValue: ContentAlignment.Right));
 
+    /// <summary>
+    /// The backing <see cref="DependencyProperty"/> for the <see cref="IsActionIconVisible"/> property.
+    /// </summary>
+    public static readonly DependencyProperty IsActionIconVisibleProperty = DependencyProperty.Register(
```

Comment:
Think name is fine, kind of maps to stuff like `NavigationView` has like [`IsPaneToggleButtonVisible`](https://learn.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.navigationview.ispanetogglebuttonvisible?view=winui-2.8).